Resources/public/js/views/ez-searchview.js

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-searchview', function (Y) {
+    "use strict";
+    
+    Y.namespace('eZ');
+
+    Y.eZ.SearchView = Y.Base.create('searchView', Y.eZ.TemplateBasedView, [], {
+        events: {
+            '.ez-search-form': {
+                'submit': '_handleFormSubmit'
```

Comment:
`_handleFormSubmit` is a very bad name, without looking at the method, I don't know what it is doing